2021/01/20 11:55:47 : Traceback (most recent call last):
  File "/opt/lampp/htdocs/v32/cartmigration/controllers/migration.py", line 665, in storage_data
    getattr(self.get_source_cart(), 'finish_storage_data')()
  File "/opt/lampp/htdocs/v32/cartmigration/models/basecart.py", line 3929, in finish_storage_data
    folder_upload = self.get_folder_upload(self._migration_id)
  File "/opt/lampp/htdocs/v32/cartmigration/models/basecart.py", line 3941, in get_folder_upload
    folder_upload = root_path + '/' + DIR_UPLOAD + '/' + get_config_ini('src', 'file', file = 'test.ini')
TypeError: must be str, not NoneType

2021/01/20 11:55:51 : orders_errors 1: Traceback (most recent call last):
  File "/opt/lampp/htdocs/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/opt/lampp/htdocs/v32/cartmigration/models/cart/custom.py", line 1168, in convert_order_export
    billing_state = get_row_from_list_by_field(orders_ext['data']['zones'], 'code', order[
KeyError: 'zones'

2021/01/20 11:55:51 : orders_errors 2: Traceback (most recent call last):
  File "/opt/lampp/htdocs/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/opt/lampp/htdocs/v32/cartmigration/models/cart/custom.py", line 1168, in convert_order_export
    billing_state = get_row_from_list_by_field(orders_ext['data']['zones'], 'code', order[
KeyError: 'zones'

2021/01/20 11:55:51 : orders_errors 3: Traceback (most recent call last):
  File "/opt/lampp/htdocs/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/opt/lampp/htdocs/v32/cartmigration/models/cart/custom.py", line 1168, in convert_order_export
    billing_state = get_row_from_list_by_field(orders_ext['data']['zones'], 'code', order[
KeyError: 'zones'

2021/01/20 11:55:51 : orders_errors 4: Traceback (most recent call last):
  File "/opt/lampp/htdocs/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/opt/lampp/htdocs/v32/cartmigration/models/cart/custom.py", line 1168, in convert_order_export
    billing_state = get_row_from_list_by_field(orders_ext['data']['zones'], 'code', order[
KeyError: 'zones'

2021/01/20 11:55:52 : orders_errors 5: Traceback (most recent call last):
  File "/opt/lampp/htdocs/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/opt/lampp/htdocs/v32/cartmigration/models/cart/custom.py", line 1168, in convert_order_export
    billing_state = get_row_from_list_by_field(orders_ext['data']['zones'], 'code', order[
KeyError: 'zones'

2021/01/20 11:55:52 : orders_errors 6: Traceback (most recent call last):
  File "/opt/lampp/htdocs/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/opt/lampp/htdocs/v32/cartmigration/models/cart/custom.py", line 1168, in convert_order_export
    billing_state = get_row_from_list_by_field(orders_ext['data']['zones'], 'code', order[
KeyError: 'zones'

2021/01/20 11:55:52 : orders_errors 7: Traceback (most recent call last):
  File "/opt/lampp/htdocs/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/opt/lampp/htdocs/v32/cartmigration/models/cart/custom.py", line 1168, in convert_order_export
    billing_state = get_row_from_list_by_field(orders_ext['data']['zones'], 'code', order[
KeyError: 'zones'

2021/01/20 11:55:52 : orders_errors 8: Traceback (most recent call last):
  File "/opt/lampp/htdocs/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/opt/lampp/htdocs/v32/cartmigration/models/cart/custom.py", line 1168, in convert_order_export
    billing_state = get_row_from_list_by_field(orders_ext['data']['zones'], 'code', order[
KeyError: 'zones'

